src/libs/rover/scheduler.cpp

```diff
@@ -651,31 +658,25 @@ void Scheduler<FloatType>::to_blueprint(conduit::Node &dataset)
       conduit::Node &n_int = dataset["fields/intensities"];
       n_int["topology"] = topo_name;
       n_int["association"] = "element";
+      n_int["units"] = "intensity units";
```

Comment:
For now this is fine. In reality, the units will be passed through the filter as arguments and they will show up here. Later on we will want to make this happen.